handlers/update_resource.go

```diff
@@ -83,3 +72,12 @@ func (d *updateResourceEntry) addToStream(id string) error {
 	}
 	return d.stream.SendMessage(string(message))
 }
+
+func (d *updateResourceEntry) compareAndUpdateResource(id string, newResource datautils.Resource, existingResource *datautils.Resource) error {
+
+	resourceForUpdate, err := newResource.CompareResource(existingResource)
+	for k, v := range resourceForUpdate {
+		log.Printf("[%s] = [%v]", k, v)
```

Comment:
Seems like this is just doing comparing and logging here now.  No update is happening, so calling this method "compareAndUpdateResource" is a bit misleading.

Reply:
Sorry, this was a midway change that I pushed up while debugging something else, it will be complete later this morning.

Reply:
@aaron-collier That's fine, would you mind labeling the PR as WIP?